packages/app-page-builder/src/editor/plugins/elements/button/ButtonSettings.tsx

```diff
@@ -75,8 +75,11 @@ const ButtonSettings = ({
         postModifyElement: updateButtonElementIcon
     });
 
-    const updateType = useCallback(value => getUpdateValue("type")(value), [getUpdateValue]);
-    const updateIcon = useCallback(value => getUpdateValue("icon.id")(value?.id), [getUpdateValue]);
+    const updateType = useCallback((value: any) => getUpdateValue("type")(value), [getUpdateValue]);
```

Comment:
I would say that `value` is a `string`.

Reply:
You are correct, but it looks like [this](https://github.com/webiny/webiny-js/pull/3788/files#diff-9d98849c0deffdf9bfad3458e47ea8fdaadeb6d2f801c26504ba4a6be1d82701R44) should also be changed to string.